cardigan/stories/global/palette/palette.stories.tsx

```diff
@@ -50,40 +78,151 @@ function hexToRgb(hex) {
     : null;
 }
 
-const colorsArr = Object.entries(themeValues.colors)
+// Code taken from https://css-tricks.com/converting-color-spaces-in-javascript/
+function rgbToHsl({ r, g, b }) {
+  // Make r, g, and b fractions of 1
+  r /= 255;
+  g /= 255;
+  b /= 255;
+
+  // Find greatest and smallest channel values
+  const cmin = Math.min(r, g, b);
+  const cmax = Math.max(r, g, b);
+  const delta = cmax - cmin;
+  let h = 0;
+  let s = 0;
+  let l = 0;
+
+  // Calculate hue
+  // No difference
+  if (delta === 0) h = 0;
+  // Red is max
+  else if (cmax === r) h = ((g - b) / delta) % 6;
+  // Green is max
+  else if (cmax === g) h = (b - r) / delta + 2;
+  // Blue is max
+  else h = (r - g) / delta + 4;
+
+  h = Math.round(h * 60);
+
+  // Make negative hues positive behind 360°
+  if (h < 0) h += 360;
+
+  // Calculate lightness
+  l = (cmax + cmin) / 2;
+
+  // Calculate saturation
+  s = delta === 0 ? 0 : delta / (1 - Math.abs(2 * l - 1));
+
+  // Multiply l and s by 100
+  s = +(s * 100).toFixed(1);
+  l = +(l * 100).toFixed(1);
+
+  return { h, s, l };
+}
+
+let objectColors: ColorsObject = {
+  core: {
+    name: 'Core',
+    description:
+      'The core colour theme is defined as black, white and yellow. This is a constant theme which persists across the product. As a result, any additional colours should be complimentary to these.',
+  },
+  accent: {
+    name: 'Accents',
+    description:
+      'Chosen to match the core yellow, accent colours are interspersed where appropriate for uses such as: defining a theme or differentiating types of content.',
+  },
+  neutral: {
+    name: 'Neutrals',
+    description:
+      'The neutral theme is used for structural page elements such as dividers and UI components. Their variable names follow Material design and font-weight inspired naming, where the "thicker" the font, the darker the grey. Some of them have equivalents in warmNeutrals, twinned through their name. They are considered equivalents because of their luminosity levels.',
+  },
+  warmNeutral: {
+    name: 'Warm Neutrals',
+    description:
+      'Warmer versions of Neutrals, they all match their equivalent in name, but with a warmer tone reminiscent of the core yellow. They are considered equivalents because of their luminosity levels.',
+  },
+  validation: {
+    name: 'Validation',
+    description:
+      'These colours should be used solely for validation purposes. We encourage the use of different shades if for other purposes.',
+  },
+};
```

Comment:
Having these descriptions are really nice for understanding why the colours are used. ✨

Reply:
Based off of Gareth's official ones, may his Wellcome spirit live on in our codebase 😄